responses.py

```diff
@@ -302,6 +327,13 @@ def _url_matches(self, url, other, match_querystring=False):
         else:
             return False
 
+    def _body_matches(self, match, request_body):
+        for matcher in match:
+            if not matcher(request_body):
```

Comment:
I wonder if this should be `body` so that if in the future we add query string or URL the keyword arguments would have succinct names.

Reply:
The problem we run into is that the Response has its own `body`. This method gets called like so:

```
        if not self._url_matches(self.url, request.url, self.match_querystring):
            return False

        if not self._body_matches(self.match, request.body):
            return False
```

So it makes sense to pass the body from the request in this way, but if it's just called `body` in the function then we end up in a bug-prone state where `body` and `self.body` are different things. Open to ideas on this.